ui/components/ui/icon/verified-icon.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function IconVerified({
```

Comment:
We can split this out as well

Reply:
Since we're not using it, I removed the file and all references to it from this branch.